content/docs/introduction/support.md

```diff
@@ -103,93 +103,40 @@ If you, the Customer, believe that your invoice or billing receipt is incorrect,
 
 ### Response times
 
-Neon aims to respond to all **paid subscription** requests in a timely manner and as soon as practically possible. Customers are prioritized based on their plan and [Severity](#severity-levels) of their issue. We only commit to responding to Customers with an Scale plan using the target response time guidelines below.
+Neon aims to respond to all [legacy Enterprise plan](https://neon.com/docs/introduction/legacy-plans#enterprise-plan-legacy) support requests in a timely manner and as soon as practically possible. Customers are prioritized based on their support plan tier and the [Severity](#severity-levels) of their issue.
 
-#### Target response times
-
-The table below outlines Neon’s guidelines for the various support tiers of our Standard support plan.
+The table below outlines Neon’s guidelines for the support tiers of our Enterprise support plan.
 
 These times relate to the time it takes Neon to respond to the Customer’s initial request. This guideline only applies when submitting a support ticket through the Neon Console.
 
-|    Severity Level     | Standard                                  | Gold                                     | Platinum                                 |
```

Comment:
Please rename Standard/Gold to Business/Production, to match the current naming.